src/daemon/rpmostreed-deployment-utils.c

```diff
@@ -855,7 +933,10 @@ advisory_equal (gconstpointer v1,
         u     advisory kind (enum DnfAdvisoryKind)
         u     advisory severity (enum RpmOstreeAdvisorySeverity)
         as    list of packages (NEVRAs) contained in the advisory
-        a{sv} additional info about advisory (none so far)
+        a{sv} additional info about advisory
+          "cve_references" -> 'a(ss)'
+            s   title
```

Comment:
I'd be a bit wary of including the title here, as it is a piece of information that is likely to be update during a CVE lifetime (see embargoed or misreported issues).

Reply:
Note that the title here comes from the RHBZs that are attached to the errata and pushed to the mirrors as part of the rest of the rpm-md. By that stage, the CVE has already had its embargo lifted and the title is more likely to be accurate by then. We're not fetching directly from RHBZ here or anything like that. This is downstream from all of that. (See e.g. the box in this section: https://fedoraproject.org/wiki/Security_Bugs#Embargoed_Security_Issues).

That said, I did play around a bit with using the API from https://www.circl.lu/services/cve-search/ since the CVE summary there is generally nicer, but it wasn't always up to date, and it would just make `status -a` too verbose. The RHBZ title is a nice compromise for a succinct description of the CVE, and then one can open the link if they want more info.

Reply:
(Marking as resolved but feel free to unresolve! :))